grepos/app/src/main/java/net/boondockradio/grepos/dao/GithubDao.java

```diff
@@ -0,0 +1,29 @@
+package net.boondockradio.grepos.dao;
+
+import net.boondockradio.grepos.api.GithubApi;
+import net.boondockradio.grepos.dto.Repositories;
+
+import rx.Observable;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.schedulers.Schedulers;
+
+public class GithubDao {
+
+    private GithubApi mGithubApi;
+
+    public GithubDao(GithubApi githubApi) {
+        mGithubApi = githubApi;
```

Comment:
Any reason to pass in the constructor?

`mGithubApi = ApiClient.getClient().create(GithubApi.class);`